pyflakes/test/test_imports.py

```diff
@@ -564,6 +563,15 @@ def bar():
         fu
         ''', m.UnusedImport, m.UndefinedName)
 
+    def test_shadowedInNestedScope(self):
+        self.flakes('''
+        import fu
+        def bar():
+            import fu
+            fu
+        fu
+        ''', m.ImportShadowedByImport)
```

Comment:
IMO it isnt the same, as imports are an external resource, and should be consistently named through the module.  I can appreciate if others think that is too much a style issue for pyflakes to take on.

However, the case I am most interested in is

```
import foo

def blah():
     import foo
     foo

foo
```

Currently that isnt caught as a redefinition because `foo` is used.  However the import in the function is superfluous, and should be removed.